packages/terra-graphs/tests/wdio/Graphs/CombiantionGraph/combination-graph-spec.js

```diff
@@ -0,0 +1,55 @@
+Terra.describeViewports('CombinationGraph', ['tiny', 'medium', 'large'], () => {
```

Comment:
Nit: typo in folder name

Reply:
File renamed in this [commit](https://github.com/cerner/terra-graphs/pull/150/commits/768d7923e4f325eaef40c8caa6ff6753069fc96e).